app/templates/suppliers/create_your_account.html

```diff
@@ -1,5 +1,4 @@
 {% extends "_base_page.html" %}
-{% import "toolkit/summary-table.html" as summary %}
```

Comment:
Weird, I'm not sure this page ever used this import.

Reply:
It is weird